xml/Menu/birthdays.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<menu>
+  <item>
+    <path>civicrm/admin/birthdays</path>
```

Comment:
There's some convention to place settings forms at `civicrm/admin/setting/<short-ext-name>`.

Reply:
This is weird. Using your suggestion leads to:
`DOMAIN/civicrm/civicrm/admin/setting/birthdays`

Removing the first `civicrm/` block leads to a crash:
```
Error message
Notice: Undefined index: component_id in CRM_Core_Menu::buildAdminLinks() (line 380 of /var/www/civicrm_env/alle/drupal/vendor/civicrm/civicrm-core/CRM/Core/Menu.php). 
```

Any idea whats wrong here?